guides/source/active_record_encryption.md

```diff
@@ -223,15 +221,15 @@ class Person
 end
 ```
 
-They will also work when combining encrypted and unencrypted data, and when configuring previous encryption schemes.
+They will also work when combining encrypted and unencrypted data,git and when configuring previous encryption schemes.
```

Comment:
could you please clarify, I don't understand this change.

Reply:
Good catch @mikdiet, that certainly looks like an oversight. Would you mind creating a PR with fix? I can't do it if not.